doc/docs.md

```diff
@@ -2527,16 +2530,25 @@ import
 in
 interface
 is
+lock
 match
 module
 mut
 none
 or
 pub
 return
+rlock
+select
+shared 
+sizeof
+static
 struct
+switch
```

Comment:
`switch` is removed with match.

Reply:
It's still reserved. I'll update this PR when your [PR](https://github.com/vlang/v/pull/6056) is merged.

Reply:
done